components/prombench/manifests/benchmark/3_prometheus-test.yaml

```diff
@@ -698,6 +698,68 @@ data:
         source_labels: [__meta_kubernetes_pod_node_name]
         target_label: nodeName
 
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: prometheus-test-restarter-scripts
+  namespace: prombench-{{ .PR_NUMBER }}
+data:
+  restarter.sh: |
+    #!/bin/sh
+    echo "Restarting prometheus"
+    kill -SIGHUP $(pidof start.sh)
+  killer.sh: |
+    #!/bin/sh
+    echo "Killing prometheus"
+    kill -9 $(pidof prometheus)
+  fetchtini.sh: |
+    #!/bin/sh
+    echo "Downloading tini init system"
+
+    if [ -z $VOLUME_DIR ]; then
+        echo "environment variables not set correctly, exiting."
+        exit 1;
+    fi
+
+    wget -O $VOLUME_DIR/tini \
+    https://github.com/krallin/tini/releases/download/v0.18.0/tini-static
+    chmod +x $VOLUME_DIR/tini
```

Comment:
we can move the tini download part to the prometheus-builder image
then run prometheus-builder for both deployments

- the release deployment should only copy tini to the required directory
- the pr deployment will work same + copy tini to the required directory

this way we won't need to run this script from k8s and reduce the number of
initContainers by 1 for pr deployment